<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editIssue</class>
 <widget class="QDialog" name="editIssue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="minimum">
      <number>1900</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>January</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>February</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>March</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>April</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>May</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>June</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>July</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>August</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>September</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>October</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>November</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>December</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="issueNumLabel">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="text">
      <string>Issue Number:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="journalNameLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Journal Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="volumeLabel">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Volume</string>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="monthLabel">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="text">
      <string>Month:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="yearLabel">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editIssue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editIssue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
